unit-tests/src/main/scala/tests/Main.scala

```diff
@@ -5,7 +5,13 @@ import java.lang.System.exit
 object Main {
   def main(args: Array[String]): Unit = {
     // tests.Discover object is code-generated in the sbt build
-    val suites  = tests.Discover.suites
+    val allSuites = tests.Discover.suites
+
+    val suites =
+      if (args.size == 1)
+        allSuites.filter(_.getClass.getSimpleName == args.head)
+      else allSuites
```

Comment:
There is a similar change in @Duhemm's PR that uses contains instead:

https://github.com/scala-native/scala-native/pull/574/files#diff-597e95ecf4ef0aabcb0550163f9588baR9

That's probably a better way to do it.